src/frontend/apps/e2e/__tests__/app-desk/menu.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { expect, test } from "@playwright/test";
+
+import { keyCloakSignIn } from "./common";
+
+test.beforeEach(async ({ page }) => {
+  await page.goto("/");
+  await keyCloakSignIn(page);
+});
+
+test.describe("Menu", () => {
+  test("checks all the elements are visible", async ({ page }) => {
+    const menu = page.locator("menu").first();
+
+    await expect(menu.getByLabel("Search button")).toBeVisible();
+    await expect(menu.getByLabel("Favoris button")).toBeVisible();
+    await expect(menu.getByLabel("Recent button")).toBeVisible();
+    await expect(menu.getByLabel("Contacts button")).toBeVisible();
+    await expect(menu.getByLabel("Groups button")).toBeVisible();
+
+    await menu.getByLabel("Search button").hover();
+    await expect(menu.getByLabel("tooltip")).toHaveText("Search");
+
+    await menu.getByLabel("Contacts button").hover();
+    await expect(menu.getByLabel("tooltip")).toHaveText("Contacts");
+  });
```

Comment:
suggestion: Could we parametrize this test? And check each label on hover? 
I have never done it with playwright, but it seems possible https://playwright.dev/docs/test-parameterize#parameterized-tests

Reply:
https://github.com/numerique-gouv/people/blob/5476ee813efb182b9994978dfab0908ffb8cca80/src/frontend/apps/e2e/__tests__/app-desk/menu.spec.ts